amplify/backend/function/expressServer/src/models/league.model.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose')
+const {Schema} = mongoose
+
+const LeagueSchema = new mongoose.Schema(
+    {
+      leagueName: { type: String, trim: true, required: true },
+      budget: { type: String, trim: true, required: true},
```

Comment:
Should this be a numerical value?

Reply:
That's a good spot. I'll change that now and update the file